etc/db_schema.xml

```diff
@@ -7,14 +7,13 @@
         <column xsi:type="int" name="store_id" nullable="false" comment="ID of store the change was issued on"/>
         <column xsi:type="smallint" name="item_type" nullable="false" comment="Type of item"/>
         <column xsi:type="varchar" name="operation_type" nullable="false" length="255" comment="Operation Type"/>
+        <column xsi:type="int" name="skip_entity_id_binding" unsigned="true" nullable="true" identity="false"
+            comment="Ignore Entity ID Binding"/>
         <constraint xsi:type="primary" referenceId="PRIMARY">
             <column name="entity_id"/>
         </constraint>
-        <constraint xsi:type="unique" referenceId="DOOFINDER_FEED_CHANGED_ITEM_ITEM_ID_STORE_ID_OPERATION_TYPE_ITEM_TYPE">
-            <column name="item_id"/>
-            <column name="store_id"/>
-            <column name="operation_type"/>
-            <column name="item_type"/>
-        </constraint>
+        <constraint xsi:type="foreign" referenceId="DOOFINDER_FEED_CHANGED_ITEM_SKIP_ENTITY_ID_BINDING_FK"
+                    table="doofinder_feed_changed_item" column="skip_entity_id_binding" 
+                    referenceTable="catalog_product_entity" referenceColumn="entity_id" onDelete="CASCADE"/>
```

Comment:
onDelete cascade?
Even if it's always null this doesn't sound good, no?

Reply:
Como bien dices realmente este onDelete cascade no tiene efectos prácticos. Se puede quitar sin problemas creo.

Lo voy a comprobar y lo subo.